tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -3915,3 +3915,42 @@ public function getFormObject($class) {
   }
 
 }
+
+/**
+ * Test SMS provider to allow for testing
+ */
+class testSMSProvider extends CRM_SMS_Provider {
```

Comment:
Zoink? `CiviUnitTestCase.php` is s strange place to put this?

Reply:
@totten Yes it is a strange place - any suggestions please?  It was in the tests/phpunit/CRM/Sms/Provider.php file.  I didn't really want to duplicate the class so needed to put it somewhere that it could be discovered by both sets of tests. CiviUnitTestCase.php seems to be the only common denominator!

Reply:
@mattwire Would something similar to `CiviTest/CiviMailUtils.php` work out?

Reply:
@mattwire @totten @mlutfy - I pushed a commit which reverts the change to this file `and` instead creates a new `tests/phpunit/CiviTest/CiviTestSMSProvider.php` file. Annoyingly(?), it seems to squash this thread / presume it is no longer needed as part of the discussion.